.platform/hooks/predeploy/06_fetch_ssm_parameters.sh

```diff
@@ -84,6 +84,7 @@ ENV_VARS=(
     "GGS_REPAIR_FORM_KEY"
     "SITE_CHECKER_FORM_KEY"
     "SITE_AUDIT_LOGS_FORM_KEY"
+    "SLACK_SIGNING_SECRET"
```

Comment:
sanity check here - not entirely sure if this is correct, since this is for eb which has been deprecated

Reply:
removed